src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -484,11 +512,20 @@ private void TestResultHandler(object sender, TestResultEventArgs e)
                 testDisplayName = string.Format("{0} [{1}]", testDisplayName, formattedDuration);
             }
 
+            var executionId = GetExecutionId(e.Result);
+            var parentExecutionId = GetParentExecutionId(e.Result);
+
+            if (parentExecutionId != null && leafExecutionIdAndTestOutcomePairDictionary.ContainsKey(parentExecutionId))
+            {
+                leafExecutionIdAndTestOutcomePairDictionary.TryRemove(parentExecutionId, out var value);
```

Comment:
>value [](start = 97, length = 5)

Are we using this anywhere?

Reply:
No, but tryRemove has that parameter as mandatory.
Any suggestions ?

Reply:
Replacing it with discard `out _` gets rid of the warning. Fixed.